templates/people/index.html.twig

```diff
@@ -9,8 +9,16 @@
         Versijuojamas JavaScript failas: <b id="versionedFile">...</b>
     </div>
 
-    <input type="text" id="name"/>
-    <span id="validation-result" data-path="{{ path('validatePerson', {'element': 'name'}) }}"></span>
+    <div class="form-inline form p-2">
+        <input type="text" id="name" placeholder="Name"/>
```

Comment:
:+1: Už `placeholder` – jau galvojau rašysiu pasiūlymą dėl [`<label for`](https://www.w3schools.com/tags/att_label_for.asp)